package.json

```diff
@@ -31,7 +31,7 @@
   "devDependencies": {
     "@fastify/pre-commit": "^2.1.0",
     "@types/node": "^22.0.0",
-    "fastify": "^5.0.0-alpha.3",
+    "fastify": "^5.0.0",
```

Comment:
Hi @Fdawgs  will this add support for fastify v5, or the work is still in progress?
Thank you.